lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -159,8 +159,7 @@ def upload(project_number, app_id, binary_path, platform)
             end
           end
           unless upload_status_response.success?
-            UI.error("It took longer than expected to process your #{binary_type}, please try again.")
```

Comment:
Curious: why crash here versus crashing in the action?

Reply:
Because it lets us crash loudly with a relevant message. If we crash when we get a nil release_id back in the action, it feels weird having a specific error message about the cause of the nil, so we'd probably want to log something here anyway.

This reminds me though, we don't want to expect nil to be returned anymore, since it only happened in this case. Removed that and updated the documentation on the method.

Reply:
👍 👏